src/ensembl/src/shared/components/layout/DocLayout.scss

```diff
@@ -0,0 +1,56 @@
+@import 'src/styles/common';
+
+$sidebarContentWidth: 320px;
+
+.DocLayout {
+  height: 100%;
+  min-height: 550px; // based on the current interface of genome browser
```

Comment:
Do we need the whole page to be scrollable? Or is it only the DocLayout that we need to be scrollable? If it is the latter then we can use `calc(100vh - 103px)` as the height and get rid of `height: 100%` in the child elements.